src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1550235650.inc.php

```diff
@@ -0,0 +1,9 @@
+<h1>Build #1550235650</h1>
+<h2>Date: 2019-02-15</h2>
+<div class="changelog">
+    - #275: Add info message for hreflang
+</div>
+<?php
+
+TCMSLogChange::addInfoMessage('hreflang: MTPageMeta provides now the array "language-alternatives" with URLs to the current page in other languages.
```

Comment:
How about something like this:

"MTPageMeta now provides an additional variable "language-alternatives" that allows to add URLs to alternative languages of the same page using hreflang meta tags. See the view MTPageMeta/standard.view.php in chameleon-shop-theme-bundle for an example."

And I'd add the message on INFO level; it's an optional improvement, so not really a TODO.

Reply:
ok, ok